cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js

```diff
@@ -32,6 +33,30 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
         },
       );
     });
+    it('should render an SMP player that is ready to play media', () => {
+      cy.window().then(win => {
+        const media =
+          getBlockData('video', win.SIMORGH_DATA.pageData) ||
+          getBlockData('version', win.SIMORGH_DATA.pageData);
+
+        if (!media) throw new Error('no media');
```

Comment:
So, this will fail if there is no media on a MAP page... which makes sense right? This test would start to fail if the media was removed from the MAP page, but I think that's OK.

Reply:
Yes :) 
And it doesn't fail when the player loads but there is an error like what I expect was a region restriction (it was in Thai).

Reply:
Although maybe, media being unavailable should be a separate test than checking the ready state on the player.

Reply:
Let's look at this in the next iteration :)